dataland-framework-toolbox/src/main/resources/org/dataland/frameworktoolbox/specific/datamodel/PrivateFrameworkDataController.kt.ftl

```diff
@@ -0,0 +1,121 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.${frameworkPackageName}
+
```

Comment:
Dieses template würde für ein weiteres private frameowkr nicht funktionieren, bzw. wenn man "Sme" umbenennen würde, würde es auch nicht mehr funktionieren.

Vorschlag: Wir sollten vllt das "Sme" Kürzel rausnehmen, und stattdessen den `framework.identifier` einfach als Variable in das template geben.  So könnte man dieses template völlig unabhängig vom Framework-Namen verwenden.